src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -194,17 +196,25 @@ def scrub(self, svc_id, deep=False):
         api_scrub = "osd deep-scrub" if str_to_bool(deep) else "osd scrub"
         CephService.send_command("mon", api_scrub, who=svc_id)
 
-    @RESTController.Resource('POST')
-    def mark_out(self, svc_id):
-        CephService.send_command('mon', 'osd out', ids=[svc_id])
-
-    @RESTController.Resource('POST')
-    def mark_in(self, svc_id):
-        CephService.send_command('mon', 'osd in', ids=[svc_id])
+    @RESTController.Resource('PUT')
```

Comment:
BTW, now I remember there was a discussion already about this endpoint and using `PUT` for updating it (https://github.com/ceph/ceph/pull/26988/files#r266064270). The thing is that `PUT` shouldn't be used for partial updates: you should always provide in the body of the request a full update for the Resource. However, that would require here that you first retrieve the enabled flags and then this endpoint is no longer atomic (you need a 2-step get+set operation). So my suggestion is to provide a [`PATCH` method](https://en.wikipedia.org/wiki/Patch_verb) instead. 

However `PATCH` payload is not defined. With JSON content types, [JSON Patch](https://en.wikipedia.org/wiki/JSON_Patch) is often used. However, I think a partial update dictionary would be enough:
```bash
# PATCH /api/osd/13/mark
{ "out": true, "lost": true }
```